src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -330,8 +330,8 @@ public function value($column)
     /**
      * Chunk the results of the query.
      *
-     * @param  int  $count
-     * @param  callable  $callback
+     * @param  int      $count
```

Comment:
You'll get hit on the fingers for these! :scream:

The formats are `@param__type__$variable` and `@return_type`.
Always the same number of spaces, no vertical alignment.